src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -33,13 +34,7 @@
     Matchable,
 )
 from sqlfluff.core.parser.segments.base import BracketedSegment
-
-from sqlfluff.core.dialects import load_raw_dialect
-
-from sqlfluff.dialects.dialect_bigquery_keywords import (
-    bigquery_reserved_keywords,
-    bigquery_unreserved_keywords,
-)
+from sqlfluff.dialects.dialect_bigquery_keywords import bigquery_reserved_keywords, bigquery_unreserved_keywords
```

Comment:
This needs to. be reverted to keep black happy:

```suggestion
from sqlfluff.dialects.dialect_bigquery_keywords import (
    bigquery_reserved_keywords,
    bigquery_unreserved_keywords,
)
```